### ISSUE TITLE
ssm = "aws:kms" - is there a way to specify SSEKMSKeyId for s3fs?

### ISSUE THREAD

**Issue by nemcidan** [danielfrg/s3contents#61]
I was not able to find whether s3contents supports "aws:kms" sse type. I was unable to find a way to specify the key ID for KMS.

in the jupyter configuration, i can specify sse type to be used:

```
from s3contents import S3ContentsManager

config = get_config()
...
config.S3ContentsManager.sse = "aws:kms"
```
which works, but files and folders are not readable. I suppose it is because KMS key ID is not specified.

After some digging in the source code, I found that [this line](https://github.com/danielfrg/s3contents/blob/d7e398c7da8836ac7579fa475bded06838e053ea/s3contents/s3_fs.py#L78) configures the SSE. is there a possibility to add the funcionality for SSEKMSKeyId to be specified all the way from the jupyter config, something like this?

```
s3_additional_kwargs["SSEKMSKeyId"] = self.sse_kms_key_id
```
example config:

```
...
config.S3ContentsManager.sse = "aws:kms"
config.S3ContentsManager.sse_kms_key_id = "id"
...
```

**Comment by danielfrg** [comment]
That's not a valid option right now but it would be super useful to have it, it should also be relatively easy to add. As you already found the code that needs to be extended.

If you change the code and test it i'll be super happy to accept a Pull Request :)

**Comment by hocanint-amzn** [comment]
We needed this so we entered https://github.com/danielfrg/s3contents/pull/77. Thanks!

**Comment by hocanint-amzn** [comment]
@danielfrg Can we cut a new release and resolve this issue? Thanks

**Comment by danielfrg** [comment]
I just released 0.3.1 that includes the fix. https://pypi.org/project/s3contents/0.3.1/

Thanks for the PR!
